test/identifier/single-statement.spec.ts

```diff
@@ -1011,84 +1011,114 @@ describe('identifier', () => {
       expect(actual).to.eql(expected);
     });
 
-    it('should identify statement using CTE with column list', () => {
-      const sql = `WITH cte_name (column1, column2) AS (
-        SELECT * FROM table
-      )
-      SELECT * FROM cte_name;`;
+    describe('identifying CTE statements', () => {
+      it('should identify statement using CTE with column list', () => {
+        const sql = `WITH cte_name (column1, column2) AS (
+          SELECT * FROM table
+        )
+        SELECT * FROM cte_name;`;
 
-      const actual = identify(sql);
-      const expected = [
-        {
-          start: 0,
-          end: 102,
-          text: sql,
-          type: 'SELECT',
-          executionType: 'LISTING',
-          parameters: [],
-        },
-      ];
+        const actual = identify(sql);
+        const expected = [
+          {
+            start: 0,
+            end: 108,
```

Comment:
Change to the `end` value here (and other existing tests) is just due to the increased indentation. No actual change to the code here, they were properly detected previously.